src/Tizen.NUI/src/public/XamlBinding/ResourceDictionary.cs

```diff
@@ -77,19 +79,25 @@ public Type MergedWith
         }
 
         /// <summary>
-        /// Gets or sets the URI of the merged resource dictionary.
+        /// Gets or sets the resource dictionary.
         /// </summary>
-        /// This will be public opened in tizen_5.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        /// Internal using, will never open.
         [EditorBrowsable(EditorBrowsableState.Never)]
         [TypeConverter(typeof(RDSourceTypeConverter))]
-        public Uri Source
+        public ResourceDictionary Source
```

Comment:
could you resolve the warning below?

**src/Tizen.NUI/src/public/XamlBinding/ResourceDictionary.cs#L87
Change 'Source' to be read-only by removing the property setter.**